src/CoreWCF.Primitives/src/CoreWCF/Security/SecuritySessionServerSettings.cs

```diff
@@ -0,0 +1,2526 @@
+            //    Message message;
+            //    if (this.TryReceive(timeout, out message))
+            //    {
+            //        return message;
+            //    }
+            //    else
+            //    {
+            //        throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new TimeoutException());
+            //    }
+            //}
+
+            //public IAsyncResult BeginReceive(AsyncCallback callback, object state)
+            //{
+            //    return this.BeginReceive(this.DefaultReceiveTimeout, callback, state);
+            //}
+
+            //public IAsyncResult BeginReceive(TimeSpan timeout, AsyncCallback callback, object state)
+            //{
+            //    return this.BeginTryReceive(timeout, callback, state);
+            //}
+
+            //public Message EndReceive(IAsyncResult result)
+            //{
+            //    Message message;
+            //    if (this.EndTryReceive(result, out message))
+            //    {
+            //        return message;
+            //    }
+            //    else
+            //    {
+            //        throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new TimeoutException());
+            //    }
+            //}
+
+            //public IAsyncResult BeginTryReceive(TimeSpan timeout, AsyncCallback callback, object state)
+            //{
+            //    return new ReceiveRequestAsyncResult(this, timeout, callback, state);
+            //}
+
+            //public bool EndTryReceive(IAsyncResult result, out Message message)
+            //{
+            //    return ReceiveRequestAsyncResult.EndAsMessage(result, out message);
+            //}
+
+            //public bool TryReceive(TimeSpan timeout, out Message message)
+            //{
+            //    TimeoutHelper timeoutHelper = new TimeoutHelper(timeout);
+            //    RequestContext requestContext;
+
+            //    if (this.TryReceiveRequest(timeoutHelper.RemainingTime(), out requestContext))
+            //    {
+            //        if (requestContext != null)
+            //        {
+            //            message = requestContext.RequestMessage;
+            //            try
+            //            {
+            //                requestContext.Close(timeoutHelper.RemainingTime());
+            //            }
+            //            catch (TimeoutException e)
+            //            {
+            //                DiagnosticUtility.TraceHandledException(e, System.Diagnostics.TraceEventType.Information);
+            //            }
+            //        }
+            //        else
+            //        {
+            //            message = null;
+            //        }
+            //        return true;
+            //    }
+            //    else
+            //    {
+            //        message = null;
+            //        return false;
+            //    }
+            //}
+
+            //public override T GetProperty<T>()
```

Comment:
Some code somewhere is requesting a FaultConverter. How is that code now going to request the fault converter?

Reply:
#263